angular/projects/components/src/public-api.ts

```diff
@@ -2,8 +2,6 @@
  * Public API Surface of components
  */
 
-// Directives
-export * from './lib/stencil-generated/components';
-
-// Package module
-export { ComponentsModule } from './lib/components.module';
+export * from './lib/components.service';
```

Comment:
Looks like the build is erroring here.

Reply:
Thanks, sorry I somehow thought I'd seen the build pass on this PR before asking for review. Clearly not.